hosts/builders/build3/configuration.nix

```diff
@@ -43,11 +44,7 @@
 
   nix = {
     settings = {
-      # avoid copying stuff over ssh
-      builders-use-substitutes = true;
```

Comment:
Not sure about this. This builder is intended to be used mostly as a remote builder. This is useful when build3 has faster internet connection than your local machine. Not sure if this is the case mostly.

Reply:
this setting is already set to true in common.nix so it's not required to be here